bcdaworker/main.go

```diff
@@ -12,20 +12,22 @@ import (
 	"strconv"
 	"syscall"
 
-	"github.com/CMSgov/bcda-app/bcda/encryption"
-
 	"github.com/jackc/pgx"
+	newrelic "github.com/newrelic/go-agent"
 	log "github.com/sirupsen/logrus"
 
 	"github.com/CMSgov/bcda-app/bcda/client"
 	"github.com/CMSgov/bcda-app/bcda/database"
+	"github.com/CMSgov/bcda-app/bcda/encryption"
 	"github.com/CMSgov/bcda-app/bcda/models"
+	"github.com/CMSgov/bcda-app/bcda/monitoring"
 	"github.com/CMSgov/bcda-app/bcda/responseutils"
 	"github.com/bgentry/que-go"
 )
 
 var (
-	qc *que.Client
+	qc  *que.Client
+	txn newrelic.Transaction
```

Comment:
yeah I see what your saying but this part can be included in the monitoring file also. But I'm not married to the idea of keeping all of our new relic references/core implementation in one place. Just a thought